### ISSUE TITLE
 ValueError: invalid literal for int() with base 10: '1.35.5.7'

### ISSUE THREAD

**Issue by ghost** [Azelphur-Servers/django-game-info#1]
After installing, migrating and 1 attempt of `update_game_info` I've received this as response in my console.

```Traceback (most recent call last):
  File "./manage.py", line 22, in <module>
    execute_from_command_line(sys.argv)
  File "/home/rafal/.env/csgodjango/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 367, in execute_from_command_line
    utility.execute()
  File "/home/rafal/.env/csgodjango/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 359, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/rafal/.env/csgodjango/local/lib/python2.7/site-packages/django/core/management/base.py", line 294, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/rafal/.env/csgodjango/local/lib/python2.7/site-packages/django/core/management/base.py", line 345, in execute
    output = self.handle(*args, **options)
  File "/home/rafal/.env/csgodjango/local/lib/python2.7/site-packages/game_info/management/commands/update_game_info.py", line 12, in handle
    server.update_info()
  File "/home/rafal/.env/csgodjango/local/lib/python2.7/site-packages/game_info/models.py", line 63, in update_info
    info_model.save()
  File "/home/rafal/.env/csgodjango/local/lib/python2.7/site-packages/django/db/models/base.py", line 796, in save
    force_update=force_update, update_fields=update_fields)
  File "/home/rafal/.env/csgodjango/local/lib/python2.7/site-packages/django/db/models/base.py", line 824, in save_base
    updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)
  File "/home/rafal/.env/csgodjango/local/lib/python2.7/site-packages/django/db/models/base.py", line 889, in _save_table
    forced_update)
  File "/home/rafal/.env/csgodjango/local/lib/python2.7/site-packages/django/db/models/base.py", line 939, in _do_update
    return filtered._update(values) > 0
  File "/home/rafal/.env/csgodjango/local/lib/python2.7/site-packages/django/db/models/query.py", line 654, in _update
    return query.get_compiler(self.db).execute_sql(CURSOR)
  File "/home/rafal/.env/csgodjango/local/lib/python2.7/site-packages/django/db/models/sql/compiler.py", line 1148, in execute_sql
    cursor = super(SQLUpdateCompiler, self).execute_sql(result_type)
  File "/home/rafal/.env/csgodjango/local/lib/python2.7/site-packages/django/db/models/sql/compiler.py", line 824, in execute_sql
    sql, params = self.as_sql()
  File "/home/rafal/.env/csgodjango/local/lib/python2.7/site-packages/django/db/models/sql/compiler.py", line 1112, in as_sql
    val = field.get_db_prep_save(val, connection=self.connection)
  File "/home/rafal/.env/csgodjango/local/lib/python2.7/site-packages/django/db/models/fields/__init__.py", line 755, in get_db_prep_save
    prepared=False)
  File "/home/rafal/.env/csgodjango/local/lib/python2.7/site-packages/django/db/models/fields/__init__.py", line 747, in get_db_prep_value
    value = self.get_prep_value(value)
  File "/home/rafal/.env/csgodjango/local/lib/python2.7/site-packages/django/db/models/fields/__init__.py", line 1832, in get_prep_value
    return int(value)
ValueError: invalid literal for int() with base 10: '1.35.5.7'
```

**Comment by Azelphur** [comment]
Yay, people are using my things :)

I pushed a commit just now which will hopefully fix this 2dcbe29aa0624c338c1011d807f9e30a5bac6cd0

You'll need to do a manage.py migrate, then run update_game_info, let me know if it works :)

**Comment by ghost** [comment]
😉 If you have more of CSGO related stuff - please let me know!

And I'll run test as soon as I get home.
